ui/open-api-ui/src/test/java/io/smallrye/openapi/ui/IndexCreatorTest.java

```diff
@@ -31,6 +32,8 @@ void testCreateDefault() throws IOException {
         assertTrue(s.contains("<img src='logo.png' alt='SmallRye OpenAPI UI'"));
         assertTrue(s.contains("dom_id: '#swagger-ui',"));
         assertTrue(s.contains("deepLinking: true,"));
+        assertTrue(s.contains("queryConfigEnabled"));
```

Comment:
If `queryConfigEnabled` is unset shouldn't it not be present in the resulting content ?